### PR TITLE
Add_DeepLearning

### LINKED ISSUE #1952: [Documentation Update] : Add Deep Learning
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Issue Description

I would like to suggest adding the Deep learning section to our website.

### Suggested Change

,

### Rationale

_No response_

### Urgency

High

### Acknowledgements

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository
- [X] I have read and followed the Contribution Guidelines
- [X] I have followed the code style guidelines of this project
- [X] I have checked for any existing open issues that my pull request may address
- [X] I have ensured that my changes do not break any existing functionality
- [X] Each contributor is allowed to create a maximum of 4 issues per day. This helps us manage and address issues efficiently
- [X] I have read the resources for guidance listed below
- [X] I have not used AI-generated content (e.g., ChatGPT, other AI tools)
- [X] I have not used content from existing sites (e.g., text, images)
- [X] I have followed all security rules and only shared trusted resources

### PR DESCRIPTION
**Pull request by Khushi-Pushkar**
Added Deep Learning language
Kindly comment if any changes needed
Fix #1952

### CODE REVIEW

**Comment by Khushi-Pushkar** [comment]
@Ajay-Dhangar Kindly review and merge

File: docs/Deep Learning/Ann.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
+---
+id: ann
```
Comment (Ajay-Dhangar) [inline docs/Deep Learning/Ann.md:2]: replace `ann` to `artificial-neural-networks` for SEO

File: docs/Deep Learning/Intro.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+---
+id: intr-dl
```
Comment (Ajay-Dhangar) [inline docs/Deep Learning/Intro.md:2]: replace `id: intr-dl` to `id: introducation-to-deep-learning`

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
update code based on suggestions

**Comment by Khushi-Pushkar** [comment]
done @Ajay-Dhangar

**Review by Ajay-Dhangar — APPROVED** [review]
done

**Pull request merged**